### ISSUE TITLE
npm warnings and errors trying to install react-pixi 6.3.0

### ISSUE THREAD

**Issue by mrbjjackson** [inlet/react-pixi#286]
### Description
Hi,
I'm trying to install React Pixi on my Gatsby.js project. I am using the latest version of Gatsby.

I've just tried to `run npm i @inlet/react-pixi` and I'm getting the following in my terminal (see below). 

It's the first time I've seen this kind of error and I'm not quite sure what it all means.

Please can you advise me on what to do?

Thanks very much.
Ben

```
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   peer react@">=16.8 || ^17.0.0" from framer-motion@3.10.5
npm WARN   node_modules/framer-motion
npm WARN     framer-motion@"^3.10.5" from the root project
npm WARN   6 more (gatsby-plugin-google-analytics, react-gsap, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"16.8.x" from react-konva@16.8.6
npm WARN node_modules/react-spring/node_modules/react-konva
npm WARN   peer react-konva@">=16.8" from @react-spring/konva@9.0.0-rc.3
npm WARN   node_modules/react-spring/node_modules/@react-spring/konva
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   peer react@">=16.8 || ^17.0.0" from framer-motion@3.10.5
npm WARN   node_modules/framer-motion
npm WARN     framer-motion@"^3.10.5" from the root project
npm WARN   6 more (gatsby-plugin-google-analytics, react-gsap, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.0.0" from react-dom@16.8.6
npm WARN node_modules/react-spring/node_modules/react-dom
npm WARN   peer react-dom@"16.8.x" from react-konva@16.8.6
npm WARN   node_modules/react-spring/node_modules/react-konva
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: konva@7.2.5
npm WARN node_modules/konva
npm WARN   peer konva@">=2.6" from @react-spring/konva@9.0.0-rc.3
npm WARN   node_modules/react-spring/node_modules/@react-spring/konva
npm WARN     @react-spring/konva@"9.0.0-rc.3" from react-spring@9.0.0-rc.3
npm WARN     node_modules/react-spring
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer konva@"^3.2.3" from react-konva@16.8.6
npm WARN node_modules/react-spring/node_modules/react-konva
npm WARN   peer react-konva@">=16.8" from @react-spring/konva@9.0.0-rc.3
npm WARN   node_modules/react-spring/node_modules/@react-spring/konva
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   peer react@">=16.8 || ^17.0.0" from framer-motion@3.10.5
npm ERR!   node_modules/framer-motion
npm ERR!     framer-motion@"^3.10.5" from the root project
npm ERR!   peer react@"^16.9.0 || ^17.0.0" from gatsby-plugin-google-analytics@3.1.0
npm ERR!   node_modules/gatsby-plugin-google-analytics
npm ERR!     gatsby-plugin-google-analytics@"^3.1.0" from the root project
npm ERR!   5 more (react-gsap, the root project, @inlet/react-pixi, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.0.0" from react-reconciler@0.20.4
npm ERR! node_modules/react-zdog/node_modules/react-reconciler
npm ERR!   react-reconciler@"^0.20.4" from react-zdog@1.0.11
npm ERR!   node_modules/react-zdog
npm ERR!     peer react-zdog@">=1.0" from @react-spring/zdog@9.0.0-rc.3
npm ERR!     node_modules/@react-spring/zdog
npm ERR!       @react-spring/zdog@"9.0.0-rc.3" from react-spring@9.0.0-rc.3
npm ERR!       node_modules/react-spring
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Steps to reproduce

1.
2.

### Additional info

- `@inlet/react-pixi` version: 6.3.0
- `React` version: "^17.0.1",
- `ReactDOM` version: "^17.0.1",
- `PIXI` version: "^6.0.0",

**Comment by inlet** [comment]
Sorry to hear that @mrbjjackson.

I guess this has something to do with the DNS lookup? Your console tells it cannot resolve modules, are you connected to the internet and/or are you on a different npm registry or behind a NAT proxy/firewall?

I don’t believe this is related to this lib, so I’ll close this issue.

**Comment by inlet** [comment]
Related https://stackoverflow.com/questions/64573177/unable-to-resolve-dependency-tree-error-when-installing-npm-packages

**Comment by inlet** [comment]
You could remove your lock file (package or yarn), optionally remove the node_modules folder and try reinstalling the packages `npm install`

**Comment by mrbjjackson** [comment]
Hi @inlet Thanks for getting back to me so quickly. I'm definitely connected to the internet etc. All other modules are installing just fine. I tried removing the node_modules and package lock file and reinstalling and I'm still getting the same error.
I have tried using the --force tag and it seems to be working so far (it hasn't broken my site). Fingers crossed the react-pixi stuff works as it should.
Thanks again!
